README.md

```diff
@@ -18,7 +20,7 @@ We love to hear from you so if you have questions, comments or find a bug in the
 
 - Open an issue on this repository
 - Tweet at us! We're [@NexmoDev on Twitter](https://twitter.com/NexmoDev)
-- Or [join the Nexmo Community Slack](https://developer.nexmo.com/community/slack)
+- Or join the [Nexmo Community on Slack](https://developer.nexmo.com/community/slack)
```

Comment:
The wording here is fine. We actually refer to it as the Community Slack.

However, the references to Nexmo do need to change to Vonage. (i.e. the twitter handle and Vonage Community Slack)

Reply:
Does Vonage Community Slack channel have the same invite link?

Reply:
The link is the same, but the Twitter account is VonageDev